From a70be96c876117fb72c6cff9dc454fb22231c8b8 Mon Sep 17 00:00:00 2001
From: "M. Vefa Bicakci" <vefa.bicakci@windriver.com>
Date: Thu, 31 Mar 2022 16:37:32 -0400
Subject: [PATCH] libbnxt.spec: Add support for rdma-core v37

This modification to libbnxt.spec applies two patches to libbnxt_re, so
that it is compatible with rdma-core v37.

Signed-off-by: M. Vefa Bicakci <vefa.bicakci@windriver.com>
---
 SPECS/libbnxt.spec | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/SPECS/libbnxt.spec b/SPECS/libbnxt.spec
index 992bda9e18ff..a509a80bdddf 100644
--- a/SPECS/libbnxt.spec
+++ b/SPECS/libbnxt.spec
@@ -18,6 +18,8 @@ BuildRequires: automake autoconf libtool
 BuildRequires: kernel%{?bt_ext}-headers >= 5.10
 
 Patch0001: 0001-Set-up-stdatomic.h-if-necessary.patch
+Patch0002: 0002-rc-compat-Add-headers-from-v37.3.patch
+Patch0003: 0003-configure.ac-Detect-rdma-core-v36-and-v37-too.patch
 
 %description
 libbnxt_re provides a device-specific userspace driver for Broadcom Netxtreme RoCE Adapters
@@ -35,6 +37,8 @@ application, which may be useful for debugging.
 %prep
 %setup -q -n %{name}-%{version}
 %patch0001 -p1
+%patch0002 -p1
+%patch0003 -p1
 
 %build
 chmod a+x autogen.sh
-- 
2.29.2

